pkg/sentry/fsimpl/gofer/gofer.go

```diff
@@ -95,7 +95,42 @@ const (
 	cacheRemoteRevalidating  = "remote_revalidating"
 )
 
-const defaultMaxCachedDentries = 1000
+const (
+	defaultMaxCachedDentries         = 1000
+	defaultMaxCachedNegativeChildren = 1000
+)
+
+// +stateify savable
+type negativeNamesCache struct {
+	// namesList stores negative names with fifo list.
+	// name stored in namesList only means it used to be negative
+	// at the moment you pushed it to the list.
+	namesList stringList
+	namesLen  uint64
+}
+
+func (cache *negativeNamesCache) Empty() bool {
```

Comment:
I think this the name should be `IsInited()`. `Empty()` gives the impression that the cache can become empty later after use, which is not the case here.

Reply:
done